data/ui/figure_settings.blp

```diff
@@ -59,14 +59,9 @@ template $GraphsFigureSettingsWindow : Adw.Window {
 
                 Adw.PreferencesGroup {
                   title: _("Axis Limits");
```

Comment:
Yeah this makes sense of course, the bottom and left axes are still there when no data is loaded. So it's a bit weird to not be able to edit them then.